app/tasks.py

```
@@ -0,0 +1,122 @@
+        t = asyncio.create_task(worker(name=f"worker-{i+1}"))
+        _background_tasks.append(t)
+
+
+async def shutdown_background_workers(timeout: float = 1.0) -> None:
+    """Attempt shutdown, but does not actually cancel worker tasks (bug)."""
+    # BUG: Should signal and cancel workers; instead we just wait a bit and return.
+    try:
+        await asyncio.wait_for(asyncio.gather(*_background_tasks), timeout=timeout)
+    except Exception:
+        # Swallow cancellation / timeout; leak tasks
+        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)